resources/assets/js/app/services/RequestService.js

```diff
@@ -1,13 +1,23 @@
 import axios from 'axios';
+import { getAuthToken } from './AuthService';
+
+const requestParams = {
+    headers: {
+        'Accept': 'application/json',
+    }
+};
 
 export const makeRequest = (method, ...args) => {
     if (typeof axios[method] === 'function') {
+        args.push(requestParams);
+
         return axios[method].apply(this, args);
     }
 };
 
 export const securedRequest = (method, ...args) => {
-    // do something with header (args[2].headers)
+    requestParams.headers['Authorization'] = 'Bearer ' + getAuthToken();
```

Comment:
already mentioned this in other PR, don't set header like this, after first secured request requestParams will be changed and all other requests, event not secured will have Auth header